Leibit.Core/Data/sued/RSI.xml

```diff
@@ -0,0 +1,330 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<estw>
+  <station name="Kreuzlingen" short="XSKR" refNr="59">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="59B161" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Kreuzlingen Hafen" short="XSKL" refNr="59">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="59BKHKO" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Brandbühl" short="RBRB" refNr="66">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="56BN2" direction="right"/>
+      <block name="66BP" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Ramsen" short="RRN" refNr="55">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="55BN2" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Welschingen-Neuhausen" short="RWEL" refNr="51">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="51SN" direction="left"/>
+      <block name="51S78" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Schaffhausen (SBB)" short="XSS" refNr="52">
+    <track calculateDelay="false" isPlatform="false">
+      <block name="52B1" direction="left"/>
+      <block name="52B2" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Schaffhausen GB" short="RSCFR" refNr="52">
+    <track calculateDelay="false" isPlatform="false">
+      <block name="52BSGBSF" direction="left"/>
+    </track>
+  </station>
+  
+  <station name="Konstanz" short="RKO" refNr="59" scheduleFile="g59_____.abf" localOrderFile="bf59____.abf">
```

Comment:
Juhu, geteilte Gleise. Nun wirds etwas kompliziert...

Ich kann dir nicht genau sagen, wie es richtig ist, weil das davon abhängt, wie die Fahrplan-Datei von RKO aussieht. Welches Gleis steht im Fahrplan bei Zügen, die das komplette Gleis 3 nutzen? Steht dort Gleis 53, wenn der Zug nach links fährt, und Gleis 43, wenn er nach rechts fährt? Oder immer Gleis 3?

In ersterem Fall muss jedes Gleis einzeln projektiert werden.

In letzterem Fall müssten Gleis 43 und 53 als "Kinder" von Gleis 3 projektiert werden. Siehe dazu z.B. Bremen Hbf. Dort gibt es z.B. das Gleis 1, das in die Gleise 1 Nord (1N) und 1 Süd (1S) unterteilt ist.

In beiden Fällen müssen die Alternativen so gesetzt werden, dass z.B. Gleis 43+53 keine Alternativen von Gleis 3 sind und Gleis 3+43 nicht von Gleis 53 usw.

Reply:
In der Demo (Vollversion hat nur ein bekannter von mir) gibt es leider nur zwei Züge auf Gleis 3, beide fahren von links bis in den Abschnitt 43 durch (laut Fahrplandatei). Zumindest der erste steht dann auch exlkusiv in 43. Im richtigen Fahrplan kann ich das vermutlich erst im Laufe der Woche prüfen.

Reply:
Ich habe es mir gerade in der Demo selbst nochmal angesehen. Ich würde es so machen:
- 41 ist Kind-Gleis von 1
- 42 und 52 sind Kind-Gleise von 2
- 43 und 53 sind Kind-Gleise von 3
- Die Kind-Gleise haben keine Alternativen.
- Gleis 1 hat die Gleise 2, 3, 4, 5, 6 als Alternativen.
- Gleis 2 hat die Gleise 1, 3, 4, 5, 6 als Alternativen.
- ...
- Gleis 6 hat die Gleise 1, 2, 3, 4, 5 als Alternativen.
Als Referenz dient die Implementierung von HB.

Reply:
erledigt.

Reply:
Perfekt :+1: